LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -68,11 +71,12 @@ public async Task<ProviderAuthenticationResult> Authenticate(string username, st
             var userManager = _applicationHost.Resolve<IUserManager>();
             User user = null;
             var ldapUser = LocateLdapUser(username);
+            var ldapUid = GetAttribute(ldapUser, UidAttr)?.StringValue;
             var ldapUsername = GetAttribute(ldapUser, UsernameAttr)?.StringValue;
             _logger.LogDebug("Setting username: {LdapUsername}", ldapUsername);
             try
             {
-                user = userManager.GetUserByName(ldapUsername);
+                user = userManager.GetUserById(UserHelper.GetLdapUser(ldapUid).LinkedJfUserId);
```

Comment:
This can throw a nullref exception that won't have any context if the ldap user doesn't have the configured UidAttr.

Reply:
Wouldn't that be caught by:
````
catch (Exception e)
            {
                _logger.LogWarning("User Manager could not find a user for LDAP User, this may not be fatal", e);
            }
```
In which case the user will be `null` and  a new user will be created if the plugin config allows it.

Reply:
If not, then please suggest how you want this to be fixed.